### ISSUE TITLE
Missing execute permission for the clickhouse pid directory

### ISSUE THREAD

**Issue by darthShadow** [ClickHouse/ClickHouse#36246]
Hi,

I was adding clickhouse to another service for monitoring it's uptime via the pid file but it looks like the pid directory doesn't allow the clickhouse group to enter the directory for reading the clickhouse pid file?

It looks like the directory is missing execute permissions for the clickhouse group which would allow the group users to actually enter the directory and read the pid file which has the correct read permissions.

```
root@ubuntu:/var/run# ls -lh /var/run/clickhouse-server
total 4.0K
-rw-r----- 1 clickhouse clickhouse 4 Apr  7 22:32 clickhouse-server.pid
root@ubuntu:/var/run# ls -lh /var/run/ | grep clickhouse-server
drwxr--r-- 2 clickhouse clickhouse   60 Apr  7 22:32 clickhouse-server
```

**Comment by darthShadow** [comment]
I have left this as a question for since I am not sure whether this behaviour is intended or a bug in how the permissions are assigned.

**Comment by darthShadow** [comment]
Bump.

**Comment by alexey-milovidov** [comment]
I also not sure what behavior should be.
Maybe simply fix it by adding `x` to the directory and that's it.

**Comment by alexey-milovidov** [comment]
Most of directories on my machine have `x` for group but not every:

```
drwx------  3 root     root     60 Apr 14 23:20 docker
drwx------  2 root     root     80 Apr 14 23:20 lvm
drwx------  5 root     root    140 Apr 15 23:37 NetworkManager
drwx------  2 root     root     40 Apr 14 23:20 udisks2
```

**Comment by alexey-milovidov** [comment]
They are also owned by root.

**Comment by darthShadow** [comment]
Yep, adding `x` to the directory allows me to read the pid file within the directory which solves my issue. I have added it to a reboot crontab for now so that everytime the directory is created, it gets the correct permissions.


I think it (`x` on directory) was just missed if the intention was to allow the group to read the pid file too.

Some other examples with the execute permissions on directory for allowing the group to read pid files:

```
drwxrwsr-x 2 haproxy    haproxy      60 Apr 15 23:43 haproxy
drwxr-xr-x 2 mysql      mysql         80 Apr 15 12:32 mysqld
drwxr-xr-x 2 redis      redis        60 Apr 15 23:43 redis
```

**Comment by dhruvanand96** [comment]
++
